dmd-frontend/src/components/App.tsx

```diff
@@ -46,7 +46,7 @@ const App = () => {
       <Topbar />
       <Container className={classes.container}>
         <Editor className={classes.editor} onSaveFunc={onEditorSave}></Editor>
-        <div className={classes.sidebar}></div>
+        {/* <div className={classes.sidebar}></div> */} 
```

Comment:
Do not commit commented code

Reply:
@trinhlaosu Please response, I won't merge the code without resolved conversation.

Reply:
I'm sorry.
I removed that commented code in source.